editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx

```diff
@@ -68,9 +70,10 @@ function Component(props: Props) {
   });
   options.append("dataset", "local-authority-district");
   options.append("dataset", "region"); // proxy for Greater London Authority (GLA) boundary
+  options.append("dataset", "title-boundary")
 
   // https://www.planning.data.gov.uk/docs#/Search%20entity
-  const root = `https://www.planning.data.gov.uk/entity.json?`;
+  const root = `https://www.planning.data.gov.uk/entity.geojson?`;
```

Comment:
By changing this request to return geojson, instead of json with a WKT `geometry` field, we can save ourselvs a WKT to GeoJSON conversion and store the title boundary as GeoJSON directly in the passport.